docs/schema/pub_txpool_full.json

```diff
@@ -0,0 +1,347 @@
+            "type": "array",
+            "minItems": 1,
+            "items": { "$ref": "#/definitions/binary32" }
+        },
+        "binary32_array64": {
+            "type": "array",
+            "minItems": 64,
+            "maxItems": 64,
+            "items": { "$ref": "#/definitions/binary32" }
+        },
+        "ringct_bulletproof": {
+            "properties": {
+                "A": { "$ref": "#/definitions/binary32" },
+                "a": { "$ref": "#/definitions/binary32" },
+                "b": { "$ref": "#/definitions/binary32" },
+                "L": { "$ref": "#/definitions/binary32_array" },
+                "mu": { "$ref": "#/definitions/binary32" },
+                "R": { "$ref": "#/definitions/binary32_array" },
+                "S": { "$ref": "#/definitions/binary32" },
+                "t": { "$ref": "#/definitions/binary32" },
+                "T1": { "$ref": "#/definitions/binary32" },
+                "T2": { "$ref": "#/definitions/binary32" },
+                "taux": { "$ref": "#/definitions/binary32" },
+                "V": { "$ref": "#/definitions/binary32_array" }
+            },
+            "required": [
+                "A", "a", "b", "L", "mu", "R", "S", "t", "T1", "T2", "taux", "V"
+            ]
+        },
+        "ringct_bulletproof_plus": {
+            "properties": {
+                "A": { "$ref": "#/definitions/binary32" },
+                "A1": { "$ref": "#/definitions/binary32" },
+                "B": { "$ref": "#/definitions/binary32" },
+                "d1": { "$ref": "#/definitions/binary32" },
+                "L": { "$ref": "#/definitions/binary32_array" },
+                "R": { "$ref": "#/definitions/binary32_array" },
+                "r1": { "$ref": "#/definitions/binary32" },
+                "s1": { "$ref": "#/definitions/binary32" },
+                "V": { "$ref": "#/definitions/binary32_array" }
+            },
+            "required": ["A", "A1", "B", "d1", "L", "R", "r1", "s1", "V"]
+        },
+        "ringct_clsags": {
+            "properties": {
+                "c1": { "$ref": "#/definitions/binary32" },
+                "D": { "$ref": "#/definitions/binary32" },
+                "s": { "$ref": "#/definitions/binary32_array" }
+            },
+            "required": ["c1", "D", "s"]
+        },
+        "ringct_ecdh_tuple": {
+            "type": "object",
+             "properties": {
+                 "amount": { "$ref": "#/definitions/binary32" },
+                 "mask": { "$ref": "#/definitions/binary32" }
+             },
+             "required": ["amount", "mask"]
+        },
+        "ringct_mlsags": {
+            "properties": {
+                "cc": { "$ref": "#/definitions/binary32" },
+                "ss": {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": { "$ref": "#/definitions/binary32_array" }
+                }
+            },
+            "required": ["c1", "D", "s"]
```

Comment:
Is there a mismatch here between `ringct_mlsags`'s properties and its required fields?

Reply:
Yup, this was a copy-paste error, and new transactions don't use mlsags so its difficult to test.